src/components/hmrc-print-link/example.njk

```diff
@@ -0,0 +1,16 @@
+{% extends "main.html" %}
+
+{% block content %}
+
+<h3>Link</h3>
+<a href="#" data-module="hmrc-print-link">Print this page</a>
+</br>
+</br>
+<h3>Input Button</h3>
+<input type="button" data-module="hmrc-print-link" value="Print this page"/>
+</br>
+</br>
+<h3>Button</h3>
+<button type="button" data-module="hmrc-print-link">Print this page</button>
+
```

Comment:
small one, I think (mostly in case someone copy pastes it) that on these we should add the govuk-link and govuk-button classes - without any font styles the VRT screenshots might not be exactly the same too I think when captured on jenkins

Reply:
a small extra bit - it might be worth one of these examples using the govukButton component so show how you add the attribute via it's params